generic-styles/content-baked.less

```diff
@@ -641,3 +649,11 @@ figure {
 .not-converted-yet {
   visibility: hidden;
 }
+
+@media screen {
+  [data-media~=screenreader] {
+    .visual-hide();
+  }
+
+  [data-media]:not([data-media~=screen],[data-media~=screenreader]) { display: none }
+}
```

Comment:
don't you also need the PDF styles here? otherwise the iframe is visible in print view.

Reply:
glad you caught that, I totally missed that in the card. Updated!

Reply:
the card was not the clearest